docs/querying/api.md

```diff
@@ -871,6 +871,8 @@ $ curl http://localhost:9090/api/v1/status/buildinfo
 
 **NOTE**: The exact returned build properties may change without notice between Prometheus versions.
 
+*New in v2.15*
```

Comment:
This was added in 14: https://github.com/prometheus/prometheus/releases/tag/v2.14.0

Reply:
Well, it is 404ing for me on this build version 2.14.0 (branch: HEAD, revision: edeb7a44cbf745f1d8be4ea6f215e79e651bfe19).

It seems the commit: https://github.com/prometheus/prometheus/commit/a85e7aac0e4b32bcc6fabbb6f4059842124bc05a was added to the 2.15.x release on the #6281 PR

Reply:
So it is, can you fix the changelog too?

Reply:
Sure, I'm gonna go ahead and assume #6281 was a feature